src/plugins/dashboard/public/application/utils/create_dashboard_container.tsx

```diff
@@ -282,7 +282,12 @@ const getEmptyScreenProps = (
             getFactory: embeddable.getEmbeddableFactory,
             notifications,
             overlays,
-            SavedObjectFinder: getSavedObjectFinder(savedObjects, uiSettings),
+            SavedObjectFinder: getSavedObjectFinder(
```

Comment:
oh sorry about. not catching this earlier. im not sure if we want to update the interface like this as it might be a breaking change for any plugin using this.

i see the saved objects plugin gets started up with the services and with it is the data plugin. potentially the same with the services.application? that way we don't need to update the interface. since i can see it already touches a bunch of plugins within OSD. But no positive any plugins outside this repo (non-OpenSearch project and OpenSearch project)

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/saved_objects/public/plugin.ts#L57

Reply:
@kavilla  I tried this approach, however I see the downstream applications are directly importing the component without using the pluginStart props. Hence I decided against it.